samples/java-protobuf-tracing/.env

```diff
@@ -0,0 +1,2 @@
+USER_SERVICE_HOST=host.docker.internal
+USER_SERVICE_PORT=8080
```

Comment:
Better to copy the .env from other samples. 

This file contains not only the main variables, but also explanations about each variable.

Reply:
sure